src/user/models/link/link-address.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { IEntity } from 'src/shared/models/entity';
+import { Column, Entity, Generated, Index } from 'typeorm';
+
+@Entity()
+export class LinkAddress extends IEntity {
+  @Column({ length: 256 })
+  existingAddress: string;
+
+  @Column({ length: 256 })
+  newAddress: string;
+
+  @Column()
+  @Generated('uuid')
+  @Index({ unique: true })
+  authentication: string;
+
+  @Column({ default: false })
+  isCompleted: boolean;
+
+  @Column({ type: 'datetime2' })
+  expiration: Date;
+
+  create(existingAddress: string, newAddress: string): this {
+    this.existingAddress = existingAddress;
+    this.newAddress = newAddress;
+
+    const tomorrow = new Date();
+    tomorrow.setDate(tomorrow.getDate() + 1);
+    this.expiration = tomorrow;
+
+    return this;
+  }
```

Comment:
How about a static create method? WDYT?

Reply:
I don't really mind, tried to use our new approach. A static function would be better, I think. As it is more readable. Will change it